ArchiSteamFarm/Bot.cs

```diff
@@ -3912,6 +3920,154 @@ internal sealed class Bot : IDisposable {
+
+			if (targetSteamMasterID == SteamID) {
+				return FormatBotResponse(Strings.BotLootingYourself);
+			}
+
+			if (!LootingSemaphore.Wait(0)) {
+				return FormatBotResponse(Strings.BotLootingFailed);
+			}
+
+			try {
+				HashSet<Steam.Item.EType> tmp = new HashSet<Steam.Item.EType>();
+				string[] modes = mode.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
+				foreach (string singleMode in modes) {
+					switch (singleMode.ToUpper()) {
+						case "C": // Not sure about shortcuts.
+						case "CARD":
+							tmp.Add(Steam.Item.EType.TradingCard);
+							break;
+						case "F":
+						case "FOIL":
+							tmp.Add(Steam.Item.EType.FoilTradingCard);
+							break;
+						case "B":
+						case "BOOSTER":
+							tmp.Add(Steam.Item.EType.BoosterPack);
+							break;
+						case "E":
+						case "EMOTICON":
+							tmp.Add(Steam.Item.EType.Emoticon);
+							break;
+						case "BA":
+						case "BACKGROUND":
+							tmp.Add(Steam.Item.EType.ProfileBackground);
+							break;
+						case "U":
+						case "UNKNOWN":
+							tmp.Add(Steam.Item.EType.Unknown);
+							break;
+						case "G":
+						case "GEMS":
+							tmp.Add(Steam.Item.EType.SteamGems);
+							break;
+						case "EV":
+						case "EVERYTHING":
+							// Needs to be kept up to date, or is there an easy way for all types?
+							tmp.Add(Steam.Item.EType.TradingCard);
+							tmp.Add(Steam.Item.EType.FoilTradingCard);
+							tmp.Add(Steam.Item.EType.BoosterPack);
+							tmp.Add(Steam.Item.EType.Emoticon);
+							tmp.Add(Steam.Item.EType.ProfileBackground);
+							tmp.Add(Steam.Item.EType.Unknown);
+							tmp.Add(Steam.Item.EType.SteamGems);
+							break;
+						default:
+							// adjust error message
+							return "Unknown mode " + singleMode + "!";
+					}
+				}
+
+				HashSet<Steam.Item> inventory = await ArchiWebHandler.GetMySteamInventory(true, tmp).ConfigureAwait(false);
+				if ((inventory == null) || (inventory.Count == 0)) {
+					return FormatBotResponse(string.Format(Strings.ErrorIsEmpty, nameof(inventory)));
+				}
+
+				if (!await ArchiWebHandler.MarkSentTrades().ConfigureAwait(false)) {
+					return FormatBotResponse(Strings.BotLootingFailed);
+				}
+
+				if (!await ArchiWebHandler.SendTradeOffer(inventory, targetSteamMasterID, BotConfig.SteamTradeToken).ConfigureAwait(false)) {
+					return FormatBotResponse(Strings.BotLootingFailed);
+				}
+
+				if (HasMobileAuthenticator) {
+					// Give Steam network some time to generate confirmations
+					await Task.Delay(3000).ConfigureAwait(false);
+					if (!await AcceptConfirmations(true, Steam.ConfirmationDetails.EType.Trade, targetSteamMasterID).ConfigureAwait(false)) {
+						return FormatBotResponse(Strings.BotLootingFailed);
+					}
+				}
+			} finally {
+				LootingSemaphore.Release();
+			}
+
+			return FormatBotResponse(Strings.BotLootingSuccess);
+		}
+
+		private static async Task<string> ResponseTransfer(ulong steamID, string botNameFrom, string mode, string botNameTo) {
```

Comment:
`botNameTo` is OK, but `botNameFrom` should follow convention and be `botNames` like everywhere else.